Ports/.port_include.sh

```diff
@@ -40,6 +40,10 @@ function run_fetch_git() {
     fi
 }
 
+function run_replace_file() {
```

Comment:
This sounds like we're replacing the file itself. Maybe something like `run_replace_in_file` or maybe `run_regex_in_file`?